src/screens/FinancialSummary/FinancialSummaries.tsx

```diff
@@ -0,0 +1,151 @@
+    },
+    searchIcon: {
+      fontSize: '20px',
+      color: theme.palette.text.primary,
+      marginTop: '-5px',
+    },
+    search: {
+      border: `1px solid ${theme.palette.primary.main}`,
+      borderRadius: '10px',
+      width: '130px',
+      height: '25px',
+      padding: '2px 10px',
+    },
+    title: {
+      marginTop: '20px',
+    },
+  });
+
+interface FinancialSummariesProps extends RouteComponentProps {
+  classes: {
+    root: string;
+    content: string;
+    payButton: string;
+    viewButton: string;
+    searchIcon: string;
+    searchContainer: string;
+    search: string;
+    title: string;
+  };
+  financialSummaries?: any;
+  location: any;
+}
+
+function FinancialSummaries(props: FinancialSummariesProps) {
+  const { classes, financialSummaries } = props;
+  let paidFinancialSummaries: any[] = [];
+  let unpaidFinancialSummaries: any[] = [];
+
+  const formatFinancialSummaries = (summaries: FinancialSummaryRecord[]) => {
+      summaries.forEach(summary => {
+        const entry = { number: summary.totalAmountBilled, label: summary.lastUpdated, unit: 'MMK'};
+        if (summary.totalCustomersBilled === summary.totalCustomersPaid) {
+          paidFinancialSummaries.push(entry);
+        } else {
+          unpaidFinancialSummaries.push(entry);
+        }
+      });
+  };
+
+  formatFinancialSummaries(financialSummaries);
+
+  const getSearch = () => {
+    return (
+      <div className={classes.searchContainer}>
+        <TextField
+          className={classes.search}
+          size="small"
+          InputProps={{
+            style: { fontSize: 14 },
+            disableUnderline: true,
+            autoComplete: 'off',
+            endAdornment: <SearchIcon className={classes.searchIcon} />,
+           }}
+          color="primary"
+        />
+      </div>
+    );
+  };
+
+  const getPaymentButtons = () => {
+    return (
+      <Button className={classes.payButton} variant="contained" color="primary" disableElevation>Pay</Button>
+    );
+  };
+
+  const getViewButtons = () => {
+    return (
+      <Button className={classes.viewButton} variant="contained" color="primary" disableElevation={true}>View</Button>
+    );
+  };
+
+  return (
+    <div className={classes.root}>
+      <BaseHeader leftIcon="backNav" title="Reports" />
+      {getSearch()}
+      <div className={classes.content}>
+        <Typography className={classes.title} variant="body1">
+          Unpaid Reports
+        </Typography>
+        {unpaidFinancialSummaries.map((report: any, index: number) => (
+          <OutlinedCardList key={index} info={[report]} primary={false} rightIcon={getPaymentButtons()} />
+        ))}
+        <Typography className={classes.title} variant="body1">
+          Paid Reports
+        </Typography>
+        {paidFinancialSummaries.map((report: any, index: number) => (
+          <OutlinedCardList key={index} info={[report]} primary={false} rightIcon={getViewButtons()} />
```

Comment:
At first I was a little confused about why we're giving `OutlinedCardList` a list of single reports (since we could just be passing all the `paidFinancialSummaries` into the `info` prop). I understand now that it's for style purposes (to make each report its own card), but we might want to clarify that via a comment (on both the unpaid and paid summaries) because otherwise it just looks like weirdly written code HAHA